src/components/oppsummeringAvKlage/OppsummeringAvKlage.tsx

```diff
@@ -26,6 +27,22 @@ const OppsummeringAvKlage = (props: { klage: Klage; klagensVedtak: Vedtak }) =>
     const { formatMessage } = useI18n({
         messages: oppsummeringMessages,
     });
+
+    const hentVurderingstekst = (klage: Klage): string => {
```

Comment:
Tror jeg heller ville kalt metoden `hentVurderingstekstId` og returnert bare id-en. Så kjørt formatMessage der den brukes. Men det er mest preferanse da

Reply:
hehe, kan endre for din skull